src/org/usfirst/frc4904/robot/commands/FlywheelSpinup.java

```diff
@@ -1,12 +1,29 @@
 package org.usfirst.frc4904.robot.commands;
 
 
+import java.util.function.Supplier;
 import org.usfirst.frc4904.robot.RobotMap;
 import org.usfirst.frc4904.robot.subsystems.Flywheel;
 import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
 
-public class FlywheelSpinup extends MotorConstant {
+public class FlywheelSpinup extends MotorConstant implements OverridableCommand {
+	protected static boolean flywheelSpeedOverride = false;
+
 	public FlywheelSpinup() {
 		super("FlywheelSpinup", RobotMap.Component.flywheel, Flywheel.SHOOTING_SPEED);
 	}
+
+	public FlywheelSpinup(Supplier<Double> speedSupplier) {
+		super(RobotMap.Component.flywheel, speedSupplier.get());
```

Comment:
This calls get at construction time, which I do not think is the intended behavior.